lib/rucio/core/rule.py

```diff
@@ -3212,25 +3213,29 @@ def __create_rule_approval_email(rule, *, session: "Session"):
     recipents = __create_recipents_list(rse_expression=rule.rse_expression, filter_={'vo': vo}, session=session)
 
     for recipent in recipents:
-        text = template.safe_substitute({'rule_id': str(rule.id),
-                                         'created_at': str(rule.created_at),
-                                         'expires_at': str(rule.expires_at),
-                                         'account': rule.account.external,
-                                         'email': get_account(account=rule.account, session=session).email,
-                                         'rse_expression': rule.rse_expression,
-                                         'comment': rule.comments,
-                                         'scope': rule.scope.external,
-                                         'name': rule.name,
-                                         'did_type': rule.did_type,
-                                         'length': '0' if did['length'] is None else str(did['length']),
-                                         'bytes': '0' if did['bytes'] is None else sizefmt(did['bytes']),
-                                         'closed': not did['open'],
-                                         'complete_rses': ', '.join(rses),
-                                         'approvers': ','.join([r[0] for r in recipents]),
-                                         'approver': recipent[1],
-                                         'target_rse': target_rse,
-                                         'free_space': free_space,
-                                         'free_space_after': free_space_after})
+        text = template.safe_substitute(
+            {
+                'rule_id': str(rule.id),
+                'created_at': str(rule.created_at),
+                'expires_at': str(rule.expires_at),
+                'account': rule.account.external,
+                'email': get_account(account=rule.account, session=session).email,
+                'rse_expression': rule.rse_expression,
+                'comment': rule.comments,
+                'scope': rule.scope.external,
+                'name': rule.name,
+                'did_type': rule.did_type,
+                'length': '0' if did['length'] is None else str(did['length']),
+                'bytes': '0' if did['bytes'] is None else sizefmt(did['bytes']),
+                'open': did.get('open', "Not Applicable"),
+                'complete_rses': ', '.join(rses),
+                'approvers': ','.join([r[0] for r in recipents]),
+                'approver': recipent[1],
+                'target_rse': target_rse,
+                'free_space': free_space,
+                'free_space_after': free_space_after
+            }
+        )
```

Comment:
This is the part where I suggest separating functional from aesthetic changes. This one-line change is completely overshadowed by everything around it.

Reply:
:sweat_smile: